hs_access_control/models/community.py

```diff
@@ -33,6 +35,33 @@ def owners(self):
                                    u2ucp__community=self,
                                    u2ucp__privilege=PrivilegeCodes.OWNER)
 
+    def groups_with_public_resources(self):
```

Comment:
consider moving this below the other properties.

Reply:
I will work with @alvacouch to have his PRs reviewed/merged, so I can merge dev. For now, his feature branches are a part of this PR.

Reply:
... as in we'll incorporate your suggestion as a part of that workflow.